tests/test_firefox_trigger_network.sikuli/test_firefox_trigger_network.py

```diff
@@ -0,0 +1,11 @@
+sys.path.append(sys.argv[2])
+import browser
+import common
+
+
+com = common.General()
```

Comment:
do you want to remove "import common" and "com = common.General()" if we are not going to use this object here?